packages/asli/asli.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Interpreter for Arm's Architecture Specification Language (ASL)"
+description: """
+Interpreter for the language ASL that Arm uses to write their ISA specifications.
+
+Currently supports reading/typechecking the specification,
+interactive execution of ASL statements and expressions,
+executing opcodes one at a time,
+loading ELF files and executing Arm binaries."""
+maintainer: "Alastair Reid <alastair.d.reid@gmail.com>"
+authors: "Alastair Reid"
+license: "BSD3"
+homepage: "https://github.com/alastairreid/asl-interpreter"
+bug-reports: "https://github.com/alastairreid/asl-interpreter/issues"
+depends: [
+  "ocaml" {>= "4.09"}
+  "linenoise"
+  "menhir"
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```

Reply:
It seems like you are using ocamlfind only during compilation and not as a library. This should be tagged as a build dependency